.taskcluster.yml

```diff
@@ -167,87 +167,10 @@ tasks:
-                            taskclusterProxy: true
-                            chainOfTrust: true
-                          env:
-                            TASK_ID: ${decision_task_id}
-                            SCHEDULER_ID: ${scheduler_id}
-                            MOBILE_HEAD_REPOSITORY: ${repository}
-                            MOBILE_HEAD_BRANCH: ${event.release.target_commitish}
-                            MOBILE_HEAD_REV: ${event.release.tag_name}
-                            MOBILE_TRIGGERED_BY: ${event.sender.login}
-                          command:
-                            - /bin/bash
-                            - --login
-                            - -cx
-                            - >-
-                              git fetch origin
-                              && git reset --hard origin/main
-                              && git submodule update --init
-                              && python tools/taskcluster/release.py \
-                                --channel nightly \
-                                --commit \
-                                --output /opt/focus-android/app/build/outputs/apk \
-                                --apk focus/release/app-focus-armeabi-v7a-release-unsigned.apk \
-                                --apk focus/release/app-focus-arm64-v8a-release-unsigned.apk \
-                                --apk klar/release/app-klar-armeabi-v7a-release-unsigned.apk \
-                                --apk klar/release/app-klar-arm64-v8a-release-unsigned.apk
-                          artifacts:
-                            public/task-graph.json:
-                              type: file
-                              path: /opt/focus-android/task-graph.json
-                              expires: ${expires_in}
-                            public/actions.json:
-                              type: file
-                              path: /opt/focus-android/actions.json
-                              expires: ${expires_in}
-                            public/parameters.yml:
-                              type: file
-                              path: /opt/focus-android/parameters.yml
-                              expires: ${expires_in}
-                        extra:
-                          cron: {$json: {$eval: 'cron'}}
-                          tasks_for: ${tasks_for}
-                        metadata:
-                          name: (Focus for Android) Focus/Klar Nightly Builds (Public)
-                          description: Decision task scheduled by cron task [${cron.task_id}](https://tools.taskcluster.net/tasks/${cron.task_id})
-                          owner: ${event.sender.login}@users.noreply.github.com
-                          source: ${repository}/raw/${event.release.tag_name}/.taskcluster.yml
-                    else:
-                      # cron commented out until we port it to taskgraph
-                      #tasks_for in ["action", "cron"]
```

Comment:
You may need to dedent the block beneath this, depending how fussy yaml is about consisent indentation. If it worked on staging it's probably fine...but if things stop firing after this is merged, it could be something to investigate.